Jenkinsfile_CNP

```diff
@@ -49,6 +49,9 @@ withPipeline(type, product, component) {
 
     after('test') {
         builder.gradle('integration')
+        builder.gradle('jacocoTestReport')
```

Comment:
do we need to run Jacoco explicitly ? sonarqube task has a dependency on jacoco

Reply:
It looks like this needs to be run explicitly.  Jacoco refuses to run otherwise because there is no src folder in the root